changelog.md

```diff
@@ -4,11 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [1.4] - 2020-03-06
+## [unreleased] - 2020-05-05
  ### Added
- - [heroku-18] update PCRE to 8.44
  - [all] add nginx-debug binary
 
+ ### Changed
+ - [heroku-18] update nginx to 1.18
+ - [heroku-18] update PCRE to 8.44
```

Comment:
It looks like the `headers-more` version increased since the last release too:
https://github.com/heroku/heroku-buildpack-nginx/pull/49/files#diff-c1594fe8a33425936bb1b25a1f142c6bR14